apps/web/src/components/EditorHeader/index.tsx

```diff
@@ -204,13 +206,16 @@ export default function EditorHeader({
         <Select
           name='provider'
           label='Provider'
-          placeholder='Select a provider'
+          placeholder='Using default provider'
           options={providerOptions}
           value={selectedProvider}
+          disabled={
+            disabledMetadataSelectors || isLoading || !providerOptions.length
+          }
```

Comment:
does the placeholder show when disabled? we should tell the user they are using the default provider if none selected

Reply:
If non selected it says "default provider" yes